lib/measures/tbd/resources/psi.rb

```diff
@@ -177,6 +177,50 @@ def complete?(s) # true/false
   end
 end
 
+##
+# Check for matching vertex pairs between edges (1" tolerance).
+# @param [Hash] e1 First edge
+# @param [Hash] e2 Second edge
+#
+# @return [Boolean] Returns true if edges share vertex pairs
+def matches?(e1, e2)
```

Comment:
Isn't each edge listed only once?  When are you going to find matching edges?

Reply:
The TBD JSON input file is useful to override default edge attributes (e.g. edge type, PSI-value). One would minimally have to provide a list of linked surfaces for this to take place. That whole bit of code allowed users to narrow down the search using the edge length (2x surfaces could share more than one common edge). However, if you consider a square-shaped skylight (one of the warehouse JSON examples, BTW), all 4x perimeter edges of the skylight would necessarily be overridden (same length). The skylight here is a weird case (why override only 1x edge?) - party walls and foundations are more suitable examples.

This recent (optional) inclusion of edge vertices narrows down the search even further, and one can now pinpoint individual edges. The `count` item of JSON edge objects is now deprecated (and soon to be removed), as all edges are exported with their vertices by default (count now always equals 1).